netplan/cli/utils.py

```diff
@@ -157,9 +157,15 @@ def is_interface_matching_name(interface, match_name):
 
 
 def is_interface_matching_driver_name(interface, match_driver):
+    driver_globs = match_driver
+    if not isinstance(driver_globs, list):
```

Comment:
nitpick: could we single out the string case rather than the list? This would allow us to provide any iterable, not just an actual list.

It's purely "theoretical" of course (especially since this function will go away in #255), hence the nitpick ;-)

```suggestion
    if isinstance(driver_globs, str):
```